RedHatInsights/rhsm-subscriptions
# ENT-4530: Splunk HEC MVP

Relates to: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/29062

Testing:

1.  Pull branch
2.  Build image and push to quay
3.  Deploy to ephemeral environments using bonfire
```
bonfire deploy rhsm-subscriptions \
--no-remove-resources=all \
-i quay.io/lburnett/rhsm=splunk_hec \
-p rhsm-subscriptions/IMAGE=quay.io/lburnett/rhsm \
-i quay.io/awood/swatch-system-conduit=latest \
-p rhsm-subscriptions/CONDUIT_IMAGE=quay.io/awood/swatch-system-conduit \
--timeout 3600
```
4.  Take note of ephemeral environment it deployed to and go see that the logs are getting sent to splunk

https://splunk.corp.redhat.com/en-US/app/search/search
`index=rh_rhsm namespace="ephemeral*"`
